src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx

```diff
@@ -0,0 +1,363 @@
+---
+title: Mobile SDK guide
+tags:
+  - Mobile monitoring
+  - API guides
+metaDescription: 'Customize your mobile agent instrumentation to send custom data and metrics to New Relic.'
+---
+
+Use the New Relic mobile SDK API calls to customize and extend the data your mobile app reports to New Relic. 
+
+While our mobile monitoring solution offers powerful out-of-the-box data collection, you may need to customize and extend the data your mobile app reports to New Relic. For example, you can use the public SDK API calls to:
+
+    * Start and stop interaction traces from events in your mobile app
+    * Record custom metrics
+    * Send custom attributes and events
+    * Track network requests from libraries not supported automatically
+    * Set custom IDs to user sessions
+
+Most of these customizations are available for both native (iOS and Android) and hybrid (React Native, Xamarin, etc.) mobile apps.
+
+## Get started [#get-started]
+
+1. Make sure you have the latest mobile agent installed.
+2. Review our guidelines for using iOS and Android SDK methods:
+
+    * [Android SDK usage](#ios-sdk-usage)
+    * [iOS SDK usage](#android-sdk-usage)
+
+3. Select a customization from the table below and follow the method's documentation.
+
+## Available customizations [#methods]
+
+The table below lists some supported SDK methods to customize your native or hybrid agent instrumentation.
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "300px" }}>
+        If you want to...
+      </th>
+
+      <th>
+        Use this method
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td id="crash-analysis">
+        Track app activity that may be helpful for troubleshooting crashes.
+      </td>
+
+      <td>
+        [Record a breadcrumb](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/record-breadcrumb)
```

Comment:
Ensure naming consistency across nav labels, hyperlinks, and api pages -- For ex: "Create an attribute"(sdk api guide) vs. "Set a custom attribute" (API page) vs. "Create custom attribute (nav)"
Also check:
- "Record breadcrumbs"
- "Start/stop interactions"
- "Set application build ID"
- "Increment an attribute count"
- "Set maximum time agent stores events in memory"
- "Return current session ID"
- "Set a custom user ID to associate with events and attributes"
- "Track failing HTTP requests"
- "Configure (enable/disable) mobile monitoring settings"

Reply:
Thanks for calling this out!